### PR TITLE
Issue #162 - Added more robust handling of APID masking in CCSDSHandler

### LINKED ISSUE #162: Add more robust handling of APID masking in CCSDSHandler
Most of the time all 11 bits of the APID are not used. To counter this issue, we should allow users to select a specific mask for the APID in the config. For example,
`XXXXX1011XXX: SomePacket`
would indicate that only bits 6-9 represent the APID for SomePacket.

### PR DESCRIPTION
**Pull request by seanlu99**
Resolves #162

### CODE REVIEW

File: ait/core/server/handler.py [hunk 1]
```diff
@@ -125,26 +131,63 @@ def handle(self, input_data):
...
         udf_length = packet_data_length - self.packet_secondary_header_length
         udf_start = primary_header_length + self.packet_secondary_header_length
         user_data_field = packet[udf_start:udf_start + udf_length + 1]
-        return pickle.dumps((packet_uid, user_data_field), 2)
\ No newline at end of file
+
+        ait.core.log.info(packet_apid)
+        ait.core.log.info(user_data_field[0])
+        ait.core.log.info(user_data_field[1])
+        ait.core.log.info(user_data_field[2])
```
Comment (aywaldron) [inline ait/core/server/handler.py]: Should all these logs be in here?

File: ait/core/server/handler.py [hunk 2]
```diff
@@ -125,26 +131,63 @@ def handle(self, input_data):
...
+        # Extract APID from packet
+        packet_apid = str(bin(int(binascii.hexlify(packet[0:2]), 16) & 0x07FF))[2:].zfill(11)
+
+        # Check if packet_apid matches with an APID in the config
+        config_apid = ''
+        for i in self.packet_types:
+            if self.comp_apid(packet_apid, i):
+                config_apid = i
+                break
+        if not config_apid:
```
Comment (aywaldron) [inline ait/core/server/handler.py]: Personally I would move the comparison to each of `self.packet_types` to your `comp_apid` method, and have it return either the `config_apid` it matches or None if it doesn't match any, then replace this chunk with 
```
config_apid = self.comp_apid(packet_apid)
```
But that's just personal preference, this also works.

Comment (seanlu99) [reply ait/core/server/handler.py]: I'll change it to that. I think it looks cleaner too.

**Comment by MJJoyce** [comment]
This needs tests added (I'll open a ticket to track) but I think it's worth getting in as-is. Thanks for the PR @seanlu99!!

**Pull request merged**